ttnn/cpp/ttnn/operations/eltwise/unary/unary.cpp

```diff
@@ -278,6 +290,39 @@ Tensor Softplus::invoke(
         optional_output_tensor);
 }
 
+Tensor Dropout::invoke(
```

Comment:
Wdyt about extracting this to its own .hpp/cpp?

Reply:
Don't want to do it right now. It is implemented the same way as any other op and uses its infra.